airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -908,3 +908,19 @@ def find_processing_job_by_name(self, processing_job_name: str) -> bool:
             if e.response['Error']['Code'] in ['ValidationException', 'ResourceNotFound']:
                 return False
             raise
+
+    def delete_model(self, model_name: str):
+        """Delete Sagemaker model
+        :param model_name: (optional) name of the model
+        :return: True if Model exists and deleted else return False
```

Comment:
In general I try to keep hook behavior as close to the underlying boto functionality as possible.  Is there a particular reason you want to return True or False here when the boto call itself does not return a value?

Reply:
Yes my intent was that the delete call should not fail when model is not present. So my initial thought was :
on first run of workflow assume the sagemaker model are not created so the workflow will always fail for any use case. So I thought of return false for ValidationException where model is not present but I got ur point to keep behavior as close to boto3 so that its more understandable. Made the changes based on that. Thanks a lot